apps/condo/domains/property/components/panels/Builder/BuildingPanelEdit.tsx

```diff
@@ -453,27 +457,27 @@ const SCROLL_CONTAINER_EDIT_PADDING = '330px'
 const MENU_COVER_MAP_WIDTH = 800
 
 const ChessBoard: React.FC<IChessBoardProps> = (props) => {
```

Comment:
can we move components to separate files, so that 1 component = 1 file ?

Reply:
@akarjakin IMO it might make sense if it were reusable components. So there just local components, that could be used once